R/unnest.R

```diff
@@ -94,8 +94,7 @@ unnest.data.frame <- function(data, ..., .drop = NA, .id = NULL,
   }
 
   nested <- dplyr::transmute(dplyr::ungroup(data), !!! quos)
-  nested_unnamed <- nested %>%
-    dplyr::mutate_if(is.list, unname)
+  nested_unnamed <- map(nested, unname_list)
```

Comment:
Oh, actually I think you can use `purrr::map_if()` — sorry for not thinking of this earlier!